src/test/java/org/cloudfoundry/promregator/endpoint/EnabledEnrichmentSingleTargetMetricsEndpointTest.java

```diff
@@ -1,135 +0,0 @@
-package org.cloudfoundry.promregator.endpoint;
-
-import java.io.IOException;
-import java.util.HashMap;
-
-import org.cloudfoundry.promregator.JUnitTestUtils;
-import org.cloudfoundry.promregator.mockServer.DefaultMetricsEndpointHttpHandler;
-import org.cloudfoundry.promregator.mockServer.MetricsEndpointMockServer;
-import org.cloudfoundry.promregator.rewrite.CFAllLabelsMetricFamilySamplesEnricher;
-import org.cloudfoundry.promregator.scanner.Instance;
-import org.cloudfoundry.promregator.scanner.ResolvedTarget;
-import org.cloudfoundry.promregator.textformat004.Parser;
-import org.junit.AfterClass;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.TestPropertySource;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-
-import io.prometheus.client.Collector.MetricFamilySamples;
-import io.prometheus.client.Collector.MetricFamilySamples.Sample;
-
-@RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(classes = LabelEnrichmentMockedMetricsEndpointSpringApplication.class)
-@TestPropertySource(locations="enabledLabelEnrichment.properties")
-public class EnabledEnrichmentSingleTargetMetricsEndpointTest {
```

Comment:
Since enrichment is removed, this test will fail, and is now removed.

Reply:
make sense.